demos/SSHRemoting/README.md

```diff
@@ -83,6 +83,25 @@ In addition you will need to enable password authentication and optionally key b
 1. Restart the sshd service
     - sudo service sshd restart
 
+## Setup on MacOS Machine
+
+1. Install the latest [PowerShell for MacOS] build
+    - Make sure SSH Remoting is enabled by following these steps:
+      + Open `System Preferences`
+      + Click on `Sharing`
+      + Check `Remote Login` - Should say `Remote Login: On`
+      + Allow access to appropriate users
+1. Edit the sshd_config file at location /private/etc/ssh/sshd_config
+    - Make sure password authentication is enabled
+      + PasswordAuthentication yes
```

Comment:
The style of these is inconsistent in this file. In the windows section, we use code style for them, while in Linux and Mac sections, we use regular text. Shall we use code style for all of them?

Reply:
I'll make them consistent with code blocks.